recipes/aiofiles/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "aiofiles" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "6c4936cea65175277183553dbc27d08b286a24ae5bd86f44fbe485dfcf77a14a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<33]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - asyncio  # [py<34]
+    - singledispatch  # [py<34]
+
+test:
+  imports:
+    - aiofiles
+
+about:
+  home: https://github.com/Tinche/aiofiles
+  summary: "File support for asyncio"
+  license: Apache 2.0
+  # license_file: LICENSE
```

Comment:
This one too, please [package the license manually](http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually) and file an issue or send a PR upstream to add it to `MANIFEST.in` ([example](https://github.com/bokeh/colorcet/pull/6)).